packages/ui-extensions/src/surfaces/admin/components/Badge/Badge.doc.ts

```diff
@@ -0,0 +1,42 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'Badge',
+  description:
+    'Use this component to inform merchants of the status of an object or of an action that’s been taken.',
+  requires: '',
+  thumbnail: 'badge-thumbnail.png',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'BadgeProps',
+      description: '',
+      type: 'BadgeProps',
+    },
+  ],
+  category: 'Components',
+  subCategory: 'Titles and text',
```

Comment:
"Titles and text" was the best idea for a subcategory I could come up with. Open to other suggestions.